Add Geospatial team community page (#1503)
NixOS/nixos-homepage: src/content/teams/13_geospatial.mdx

---
name: Geospatial Team
description: Making Nix and NixOS the first choice for users of geospatial software
members:
  - name: Ivan Mincik
    discourse: imincik
    title:
  - name: Nikolay Korotkiy
    discourse: sikmir

>     title:

Author: ```suggestion
    title:
  - name: Will Cohen
    discourse: willcohen
    title:
```

contact:
  - name: Discourse

>     href: https://discourse.nixos.org/t/week-in-geospatial-team/37035

Reviewer: ```suggestion
    href: https://discourse.nixos.org/t/creating-nix-geospatial-team/23454
```

Author: ```suggestion
    href: https://discourse.nixos.org/t/creating-nix-geospatial-team/23454/9
```

That's the post where you make the announcement

  - name: Matrix
    href: https://matrix.to/#/#geospatial:nixos.org
  - name: GitHub project board

>     href: https://github.com/orgs/NixOS/projects/47/views/1

Reviewer: ```suggestion
    href: https://github.com/orgs/NixOS/projects/47
```

---

> The geospatial team is responsible for everything related to geospatial software in [Nixpkgs](https://github.com/NixOS/nixpkgs).

Author: ```suggestion
The Geospatial Team is responsible for everything related to geospatial software in [Nixpkgs](https://github.com/NixOS/nixpkgs).
```

We're keeping geospatial packages and modules up to date and we package new software regularly.

If you like [what we do](https://github.com/orgs/NixOS/projects/47), please let us know.

> If you want to join the Geospatial Team, add yourself to the geospatial team in the [Nixpkgs team list](https://github.com/NixOS/nixpkgs/blob/master/maintainers/team-list.nix).

Author: ```suggestion
If you want to join the Geospatial Team, add yourself to the `geospatial` team in the [Nixpkgs team list](https://github.com/NixOS/nixpkgs/blob/master/maintainers/team-list.nix).
```

---
name: Geospatial Team
description: Making Nix and NixOS the first choice for users of geospatial software
members:
  - name: Ivan Mincik
    discourse: imincik
    title: Team lead
  - name:  Nikolay Korotkiy
    discourse: sikmir
    title:
contact:
  - name: Discourse
    href: https://discourse.nixos.org/t/week-in-geospatial-team/37035
  - name: Matrix
    href: https://matrix.to/#/#geospatial:nixos.org
  - name: GitHub project board
    href: https://github.com/orgs/NixOS/projects/47/views/1
---

The geospatial team is responsible for everything related to geospatial software in [Nixpkgs](https://github.com/NixOS/nixpkgs).

We're keeping widely used packages and modules up to date, and package new software regularly.

> Please ping us if you need help with reviews or build failures.

Author: ```suggestion
```

If you like [what we do](https://github.com/orgs/NixOS/projects/47/views/5?filterQuery=status%3A%22%E2%9C%85+Done%22), please let us know.

If you want to join the geospatial team, add yourself to the geospatial team in the [Nixpkgs team list](https://github.com/NixOS/nixpkgs/blob/master/maintainers/team-list.nix).

> Create PR with short introduction of yourself and assign it to one of the other members.

Author: ```suggestion
Create pull request with short introduction of yourself and assign it to one of the other members.
```
